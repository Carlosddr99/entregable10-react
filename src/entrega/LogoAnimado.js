import { keyframes } from "styled-components";
import styled from "styled-components";
import logo from '../logo.svg';
import '../App.css';

const rotate = keyframes`
    from {
        transform: rotate(0deg);
    }
    to {
        transform: rotate(360deg);
    }
`;

const ContenedorAnimado = styled.div`
    display: inline-block;
    animation: ${rotate} 2s linear infinite;
    padding: 2rem 1rem;
    font-size: 1.2rem;
`;



export default function LogoAnimado(){

    return(
        <ContenedorAnimado>
            <img src={logo} className="App-logo" alt="logo"/>
        </ContenedorAnimado>
    )
}

//        <img src={logo}  />
