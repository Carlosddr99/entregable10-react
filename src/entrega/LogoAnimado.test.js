import renderer from 'react-test-renderer';
import LogoAnimado from './LogoAnimado';

it('render logo', () => {
    const tree = renderer
    .create(<LogoAnimado />)
    .toJSON();
    expect(tree).toMatchSnapshot();
});