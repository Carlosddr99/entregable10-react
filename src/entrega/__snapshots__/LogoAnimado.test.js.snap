// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render logo 1`] = `
<div
  className="sc-beqWaB hcycSS"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
</div>
`;
